pkg/scheduler/internal/cache/cache.go

```diff
@@ -638,6 +639,7 @@ func (cache *schedulerCache) cleanupExpiredAssumedPods() {
 func (cache *schedulerCache) cleanupAssumedPods(now time.Time) {
 	cache.mu.Lock()
 	defer cache.mu.Unlock()
+	defer cache.updateMetrics()
```

Comment:
nit: To make it more explicit, I would rename(or comment) `cleanupExpiredAssumedPods` to `cleanupExpiredAssumedPodsAndUpdateMetrics`.

Reply:
I don't think it's necessary to make it explicit in the function name that it updates the metrics, I know there are many other functions that report metrics as a side-effect that don't have it in the name. but I'll update the comment so it's documented